src/Domain/DonationNotifier.php

```diff
@@ -89,12 +89,6 @@ public static function emailMessageForCollectedDonation(Donation $donation): Ema
 
     public function notifyDonorOfDonationSuccess(Donation $donation): void
     {
-        if ($donation->isRegularGiving()) {
-            // Regular giving donors get an email confirming the setup of the mandate, but not an email for
-            // each individual donation.
-            return;
-        }
-
```

Comment:
I think when we're manually choosing to (re)send a donation email there's no need to exclude regular giving donations, so this logic is moved to `DonationRepository::updateDonationStatusFromSucessfulCharge`